opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -187,6 +191,8 @@ class SimulatorFullyImplicitBlackoilEbos
             ebosSimulator_.model().addAuxiliaryModule(auxMod);
         }
 
+        AquiferModel aquifer_model(ebosSimulator_, model_param_, terminal_output_);
```

Comment:
A question here. Does `Aquifer` use any results from the `Aquifer` in the previous report step? I mean, between report steps, does any information/results from previous time step transfer to the current time step? If not, it is possible the aquifer model can be created within the `BlackoilModelEbos` directly without being an argument pass into `BlackoilModelEbos`. Then the changes in this file should be removed.

Reply:
Now I see `W_flux_` is one thing transferred between time steps.